zulipterminal/ui_tools/buttons.py

```diff
@@ -244,6 +245,12 @@ def _narrow_with_compose(self, button: Any) -> None:
             recipient_user_ids=[self.user_id]
         )
 
+    def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
+        # Shows user info on pressing 's'
```

Comment:
This comment seems wrong - and likely unnecessary?

Reply:
Oops. That's not required. Thanks for pointing out :+1: